### PR TITLE
Add ssh-session plugin

### PR DESCRIPTION
**Pull request by AbelAnaya**
This plugin shows ssh connection information of active TMUX pane in the status line.

The default format is:
`user@hostname`

It can be configured to show also the connection port via `@dracula-show-ssh-session-port true`
`user@hostname:port`

When no ssh session is active it will show current user and hostname of local machine i.e: `whoami@hostname`

Color can be configured as any other dracula plugin. Default background is green and foregraund dark_gray.

Useful when user@hostname is not present in the terminal prompt or if working inside a terminal application e.g; vim, htop, etc.

Plugin appearence:
image

### CODE REVIEW

File: scripts/ssh_session.sh [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  local port=$(echo $1|grep -Eo '\-p\s*([0-9]+)'|sed 's/-p\s*//')
+
+  if [ -z $port ]; then
+    local port=22
+  fi
+
+  echo $port
+}
+
+get_ssh_user() {
```
Comment (ethancedwards8) [inline scripts/ssh_session.sh:19]: Could this function be cleaned up a bit? Looks like a lot of duplicate code.

**Review by ethancedwards8 — CHANGES_REQUESTED** [review]
Other than the function that needs to be a bit cleaner, it looks good. Thank you for your contribution!

File: scripts/ssh_session.sh (hunk 1 above)
Comment (AbelAnaya) [reply scripts/ssh_session.sh]: Function has been cleaned up and documented, thank you for pointing it out

**Comment by AbelAnaya** [comment]
Thank you for your feedback, function has been cleaned up and documented.

**Review by ethancedwards8 — APPROVED** [review]

**Pull request merged**